Core.UnitTests/CodeGeneration/ReflectionEmit/ModuleBuilderFactoryTest.cs

```diff
@@ -100,20 +100,38 @@ public void CreateModuleBuilder_StrongNamed_ProvidedKey ()
     }
 
     private void CheckAdapterAndSaveToDiskBehavior (IModuleBuilder moduleBuilder, string assemblyDirectory, byte[] expectedPublicKey = null)
+    {
+      var assemblyPath = Path.Combine (assemblyDirectory, c_assemblyFileName);
+      var pdbPath = Path.Combine (assemblyDirectory, c_pdbFileName);
+      var assemblyBuilderAdapter = (AssemblyBuilderAdapter) moduleBuilder.AssemblyBuilder;
+
+      CheckAdapterBehavior (moduleBuilder, expectedPublicKey, assemblyBuilderAdapter, assemblyPath, pdbPath);
```

Comment:
The thought behind the rewrite was that there's only two method calls left and the CheckAdapterAndSaveToDiskBehavior  can be inlined.